payjoin/src/version.rs

```diff
@@ -0,0 +1,37 @@
+use core::fmt;
+
+use serde::{Serialize, Serializer};
+
+/// The Payjoin version
+#[repr(u8)]
+#[derive(Clone, Copy, PartialEq, Eq)]
+pub enum Version {
+    One = 1,
+    Two = 2,
```

Comment:
Let's document the meaning of each version, something like:

```suggestion
    /// BIP78 Payjoin
    One = 1,
    /// BIP77 Async Payjoin
    Two = 2,
```

Reply:
done 👍🏼